locales/en/app.ftl

```diff
@@ -866,8 +866,13 @@ ad-unit-6-before-you-complete = Before you complete that next signup, use an ema
 
 # “account” can be localized, “Firefox” must be treated as a brand,
 # and kept in English.
+# Deprecated - to be replaced by -brand-mozilla-account
 -brand-fx-account = Firefox account
 
+# “account” can be localized, “Mozilla” must be treated as a brand,
+# and kept in English.
+-brand-mozilla-account = { -brand-mozilla } account
```

Comment:
As per previous comment, please use Mozilla, not a term reference within the term definition.